crates/prover/src/components/mod.rs

```diff
@@ -94,8 +101,17 @@ impl Claim {
         let (merkle_claim, merkle_trace, merkle_interaction_claim_data) =
             merkle::Claim::write_trace::<MC, MockHasher>(&input.merkle_trees);
 
+        // Write clock update trace
+        let (clock_update_claim, clock_update_trace, clock_update_interaction_claim_data) =
+            clock_update::Claim::write_trace(&input.memory.clock_update_data);
+
         // Write range_check components
-        let range_check_data = opcodes_interaction_claim_data.range_check_20();
+        let opcodes_range_check_data = opcodes_interaction_claim_data.range_check_20();
+        let clock_update_range_check_data = clock_update_interaction_claim_data
+            .lookup_data
+            .range_check_20[0]
+            .par_iter();
```

Comment:
maybe add a function `range_check_20` to the `InteractionClaimData` to move this there and keep the code simpler here